benchmark/src/__tests__/mocha.opts

```diff
@@ -1,2 +0,0 @@
---recursive
---require source-map-support/register
```

Comment:
`source-map-support/register` is also deleted?

Reply:
We use `lb-mocha` which already has source-map registered.